web/styles/compose.css

```diff
@@ -152,8 +152,8 @@
     opacity: 0.8;
     font-size: 85%;
 
-    & code {
-        display: inline-block;
+    & > * {
```

Comment:
We don't want to use wildcard selectors, see for example https://chat.zulip.org/#narrow/stream/6-frontend/topic/CSS.20selector.20performance/near/1845719.

Probably whatever element needs this display field should just have a proper CSS class.

Reply:
Read the thread more carefully; anything without a CSS class on the last item is going to have this problem; expanding the `*` doesn't help very much.

Reply:
But I can't add classes to the elements inside `autocomplete_secondary` as it markdown rendered. Maybe I can add these CSS changes to specifically `.autocomplete_secondary.rendered_markdown`? Can that solve the problem?